### PR TITLE
Reformatted using black

### PR DESCRIPTION
**Pull request by theilmbh**

### CODE REVIEW

**Comment by MarvinT** [comment]
I dig it.

I guess I'm partial to a bit longer lines but the whole point of black is that it's opinionated so I don't have to be.

Want to add some badges to the readme?

[License](https://github.com/gentnerlab/glab-common-py/blob/master/LICENSE)
[Code style: black](https://github.com/ambv/black)

```
<p align="center">
[![License](https://img.shields.io/badge/License-BSD%203--Clause-blue.svg)](https://github.com/gentnerlab/glab-common-py/blob/master/LICENSE)
[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/ambv/black)
</p>
```

File: README.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Gentner Lab Common
 ====================
 
+<p align="center">
```
Comment (MarvinT) [inline README.md]:
```suggestion
```
guess that doesn't work...

File: README.md [hunk 2]
```diff
@@ -1,6 +1,11 @@
...
+[![License](https://img.shields.io/badge/License-BSD%203--Clause-blue.svg)](https://github.com/gentnerlab/glab-common-py/blob/master/LICENSE)
+[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/ambv/black)
+</p>
```
Comment (MarvinT) [inline README.md]:
```suggestion
```

**Comment by theilmbh** [comment]
lol.  Fixed it, go ahead and merge

**Pull request merged**